Riot/Modules/GlobalSearch/DataSources/UnifiedSearchRecentsDataSource.m

```diff
@@ -69,74 +90,16 @@ - (void)setHideRecents:(BOOL)hideRecents
 
 #pragma mark - UITableViewDataSource
 
-- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
```

Comment:
Since we do all the sections calculations in `makeDataSourceSections`, there is no need to override `numberOfSections` anymore, because the superclass will just call `makeDataSourceSections` where all the changes are

Reply:
👍